examples/basic/tidb-cluster.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: pingcap.com/v1alpha1
+kind: TidbCluster
+metadata:
+  name: basic
+spec:
+  version: v3.0.8
+  timezone: UTC
+  pvReclaimPolicy: Delete
+  pd:
+    baseImage: pingcap/pd
+    replicas: 3
+    requests:
+      storage: "1Gi"
+    config: {}
```

Comment:
in my test, `config: {}` can not be omitted, is this expected?

Reply:
Yes, it could only be omitted after defaulting wehook enabled